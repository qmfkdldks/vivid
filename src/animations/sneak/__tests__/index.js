import React from "react";
import { shallow } from "enzyme";
import Sneak from "../index";

describe("Sneak", () => {
  it("should render successfully", () => {
    const component = shallow(<Sneak>Hide and sneak</Sneak>);
    expect(component).toMatchSnapshot();
  });
});
