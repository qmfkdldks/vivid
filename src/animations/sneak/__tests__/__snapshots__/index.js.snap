// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sneak should render successfully 1`] = `
<Styled(Component)
  className=""
>
  <Styled(Component)
    animate={
      Object {
        "paddingLeft": Array [
          42,
          42,
          42,
          42,
          42,
          42,
          42,
          42,
          42,
          42,
          42,
          42,
          42,
          42,
          42,
          42,
          42,
          42,
          42,
          42,
          42,
          42,
          42,
          42,
          42,
          42,
          42,
          42,
          42,
          42,
          42,
          42,
          42,
          42,
          42,
          42,
          42,
          42,
          42,
          42,
          42,
          42,
        ],
        "rotate": Array [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          10,
          0,
          0,
        ],
        "scaleX": Array [
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1.2,
          0.8,
          1.05,
          1,
          1,
          1,
          1.2,
          0.8,
          1.05,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
        ],
        "scaleY": Array [
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          0.7,
          1.2,
          0.95,
          1,
          1,
          1,
          0.7,
          1.2,
          0.95,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
          1,
        ],
        "transition": Object {
          "duration": 3,
          "loop": Infinity,
          "repeatDelay": 3,
        },
        "x": Array [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          -15,
          -15,
          -15,
          -25,
          -25,
          -25,
          -40,
          -40,
          -40,
          -50,
          -50,
          -50,
          -50,
          -50,
          -50,
          -50,
          -50,
          -50,
          -50,
          -50,
          -50,
          -50,
          -50,
          -50,
          -60,
        ],
        "y": Array [
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          -30,
          0,
          -5,
          0,
          0,
          0,
          0,
          -30,
          0,
          -5,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
          0,
        ],
      }
    }
  >
    H
     
  </Styled(Component)>
  <span>
    ide and sneak
  </span>
</Styled(Component)>
`;
